vectordatasource/transform.py

```diff
@@ -4879,6 +4997,107 @@ def _normalize_kr_netref(network, ref):
     return network, ref
 
 
+def _normalize_no_netref(network, ref):
+    prefix, number = _splitref(ref)
+
+    if prefix == 'Rv':
+        network = 'NO:Riksvei'
+        ref = number
+
+    elif prefix == 'Fv':
+        network = 'NO:Fylkesvei'
+        ref = number
+
+    elif prefix == 'E':
+        network = 'e-road'
+        ref = 'E ' + number.lstrip('0')
+
+    elif prefix == 'Ring':
+        network = 'NO:oslo:ring'
+        ref = 'Ring ' + number
```

Comment:
I'm dubious about this one for ref `Ring #` as shield_text...

Reply:
It appears to be shown on highway signage as ["Ring 3" in black on a white background inside a rounded lozenge](https://upload.wikimedia.org/wikipedia/commons/b/b3/Ring_3_and_E6_Oslo_Quevaal.jpg), like this:

![Ring 3 shield](https://upload.wikimedia.org/wikipedia/commons/4/4c/Vei_Ring3.svg)